packages/stable/src/api/geospatial/types.ts

```diff
@@ -269,3 +269,38 @@ export interface GeospatialCRSResponse
 export interface GeospatialSridId {
   srid: number;
 }
+
+export type GeospatialComputeFunction =
+  | {
+      stTransform: GeospatialGeometryTransformComputeFunction;
+    };
+
+export interface GeospatialGeometryTransformComputeFunction {
+  geospatialGeometryComputeFunction: GeospatialGeometryComputeFunction;
```

Comment:
suggestion(blocking): I think the key should be just `geometry`.

Reply:
Oh you're right, will update